integration/test/dashboard_json_test.go

```diff
@@ -104,14 +108,29 @@ func (suite *DashboardJsonTestSuite) TestJsonExpression() {
 					}
 				}
 
-				if !HasDataInDB(counter) {
+				//Test for Rest
+				query := counter + "{datacenter=~\"" + strings.Join(restDataCollectors[:], "|") + "\"}"
```

Comment:
no need to reslice restDataCollectors, this should work too `strings.Join(restDataCollectors, "|")`